### PR TITLE
docs(Storybook): Update design-guide.mdx

### PR DESCRIPTION
**Pull request by PiddyD**
Updated Design page

### CODE REVIEW

File: packages/react-component-library/src/intro/design-guide.mdx [hunk 1]
```diff
@@ -2,33 +2,170 @@ import { Meta, Source } from '@storybook/blocks'
...
 
+### Icon Library
 
-#### [Spacing](?path=/docs/tokens-spacing--docs)
-Spacing is a key (and often overlooked) aspect of ensuring your application is easy to read and understand. Too little whitespace will cause components to become visually cramped, therefore decreasing readability. The general rule is to start with whitespace that is too large for an element and then decrease it until it is at an appropriate size. It is far better to have too much whitespace than too little.
+A library of approved iconography for clear visual communication.
 
-#### [Typography](?path=/docs/tokens-typography--docs)
-All Royal Navy Design System applications are set in Lato
+[Download RNDS Axure Icon Library](https://www.)
```
Comment (markhigham) [inline packages/react-component-library/src/intro/design-guide.mdx]: Looks like a broken link?

File: packages/react-component-library/src/intro/design-guide.mdx [hunk 2]
```diff
@@ -2,33 +2,170 @@ import { Meta, Source } from '@storybook/blocks'
...
 
 
----
+## Installing Axure libraries
```
Comment (markhigham) [inline packages/react-component-library/src/intro/design-guide.mdx]: there is a funky non-visible character here which prevents the heading from being rendered properly

File: packages/react-component-library/src/intro/design-guide.mdx [hunk 3]
```diff
@@ -2,33 +2,170 @@ import { Meta, Source } from '@storybook/blocks'
...
+
+1. **Download** from the links above.
+2. **Double-click** on the file you downloaded.
+3. Axure will ask where you want to **save** the library. Commonly you might choose *'Documents > Axure > Libraries'*
+4. The library will automatically be available under the **Libraries pane**. This is near the middle left-hand side area of the screen.
+
+
+## Using RNDS library in a design file
```
Comment (markhigham) [inline packages/react-component-library/src/intro/design-guide.mdx]: same here is it a unicode whitespace character?

**Review by markhigham — CHANGES_REQUESTED** [review]
Nice work. Some red tape to take care of too.
The PR description needs some attention. Filling out the boilerplate text etc

File: packages/react-component-library/src/intro/design-guide.mdx [hunk 4]
```diff
@@ -2,33 +2,170 @@ import { Meta, Source } from '@storybook/blocks'
...
-Here’s a checklist of everything you need to get started:
+Designers can use our Axure libraries to quickly prototype, test and validate ideas with users. There are 2 libraries:
 
-## Design tokens
-The Royal Navy Design System provides tokens for colour, typography, spacing and much more. Tokens make it easy to ensure consistency across defence applications.
+### Design Component library
 
-### The case for using tokens
-The number one reason is consistency<sup>[1](#references)</sup>: the palette of `danger` is the same everywhere. End-users of Royal Navy applications highly benefit from consistency across interfaces, where the colour of danger may save lives.
+Contains RNDS Widget with ready to use interactivity.
```
Comment (m7kvqbe1) [inline packages/react-component-library/src/intro/design-guide.mdx]: Should this be widgets plural?

**Review by markhigham — APPROVED** [review]

**Review by m7kvqbe1 — APPROVED** [review]
LGTM

**Pull request merged**